api_server/app/__init__.py

```diff
@@ -64,6 +65,19 @@ def create_app(config_name=ApiConstants.DEVELOPMENT_CONFIG.value) -> FastAPI:
     app_route_includer(app)
     # Adding exceptions handlers.
     app_exception_handler(app)
+    # Allow CORS
+    origins = [
+        "http://localhost:4500",
+        "http://localhost:3030",
+        "http://localhost:3000",
```

Comment:
Can this not be hard coded, and loaded from .env file?

Reply:
Done